src/less/tooltip/base/tooltip.less

```diff
@@ -69,11 +69,13 @@ button.tooltip__close {
     .pointer-top-right();
 }
 
-.tooltip__pointer--bottom-left {
+.tooltip__pointer--bottom-left,
+span.tooltip__pointer--bottom-left {
     .pointer-bottom-left();
 }
 
-.tooltip__pointer--bottom {
+.tooltip__pointer--bottom,
+span.tooltip__pointer--bottom {
```

Comment:
Hmm, it doesn't seem right to be hammering it with more specificity (and on just two of the pointer modifiers, not all of them). I'll take a look and try and see what changed between v10 and v11. I did a big refactor to try and normalise ds4 and ds6 so I probably broke something then.

Reply:
I've been trying to figure out where this override comes from:

![Screen Shot 2021-04-13 at 8 28 34 AM](https://user-images.githubusercontent.com/38065/114579049-59927380-9c32-11eb-9c3d-342503a85a02.png)

The cascade order of the source looks good, so I suspected some overly aggressive minification, and yup when I disable minification things appears as expected:

![Screen Shot 2021-04-13 at 9 05 45 AM](https://user-images.githubusercontent.com/38065/114584480-90b75380-9c37-11eb-8685-954cc5322a3b.png)

Reply:
Yeah I can look into that!

Reply:
Sorry, this is the line I meant to point to: https://github.com/eBay/skin/blob/master/gulpfile.js#L59